### PR TITLE
snapshot br: restore execution plan bindings

### PR DESCRIPTION
**Pull request by Oreoxmt**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

 
- This PR is translated from: https://github.com/pingcap/docs-cn/pull/14926
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/br-snapshot-manual.md [hunk 1]
```diff
@@ -215,6 +218,39 @@ br restore full \
     --log-file restorefull.log
 ```
 
+### Restore execution plan bindings from the `mysql` schema
+
+To restore execution plan bindings of the original cluster, you can run the `br restore full` command, include `--with-sys-table`, and set the `mysql` schema to restore using `--filter` or `-f`.
```
Comment (qiancai) [inline br/br-snapshot-manual.md:223]:
```suggestion
To restore execution plan bindings of a cluster, you can run the `br restore full` command, including the `--with-sys-table` option and also the `--filter` or `-f` option to specify the `mysql` schema to be restored.
```

File: br/br-snapshot-manual.md [hunk 2]
```diff
@@ -215,6 +218,39 @@ br restore full \
...
+    --log-file restore_system_table.log
+```
+
+After the restore is completed, you can confirm the execution plan binding information with [`SHOW GLOBAL BINDINGS`](/sql-statements/sql-statement-show-bindings.md):
+
+```sql
+SHOW GLOBAL BINDINGS;
+```
+
+The dynamic loading of execution plan bindings after the restore is still under optimization (related issues are [#46527](https://github.com/pingcap/tidb/issues/46527) and [#46528](https://github.com/pingcap/tidb/issues/46528)). You need to manually reload the execution plan bindings.
```
Comment (qiancai) [inline br/br-snapshot-manual.md:243]:
```suggestion
The dynamic loading of execution plan bindings after the restore is still undergoing optimization (related issues are [#46527](https://github.com/pingcap/tidb/issues/46527) and [#46528](https://github.com/pingcap/tidb/issues/46528)). You need to manually reload the execution plan bindings after the restore.
```

File: br/br-snapshot-manual.md [hunk 3]
```diff
@@ -215,6 +218,39 @@ br restore full \
...
+
+```sql
+-- Ensure that the mysql.bind_info table has only one record for builtin_pseudo_sql_for_bind_lock. If there are more than one record, you need to manually delete them.
```
Comment (qiancai) [inline br/br-snapshot-manual.md:246]:
```suggestion
-- Ensure that the mysql.bind_info table has only one record for builtin_pseudo_sql_for_bind_lock. If there are more records, you need to manually delete them.
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Review by benmaoer — APPROVED** [review]
LGTM

**Pull request merged**